test/Falco.Tests/Falco.Tests.fsproj

```diff
@@ -20,6 +20,7 @@
     <Compile Include="AuthTests.fs" />
     <Compile Include="RequestTests.fs" />
     <Compile Include="RequestHandlersTests.fs" />
+    <Compile Include="Extensions/ResponseTests.Extensions.fs" />

```

Comment:
I would rather we didn't add this file, and stick with the inline Recevied call.

Reply:
I didn't want to expose details what does it mean that we have temporary/permanent redirect :+1: I will change it and leave it inlined!

Reply:
Done! :rocket: